sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -1181,13 +1175,13 @@ public partial class MetricSeriesDefinition
     }
     public partial class MetricSeriesGroupDetectionCondition : Azure.AI.MetricsAdvisor.Models.MetricWholeSeriesDetectionCondition
     {
-        public MetricSeriesGroupDetectionCondition() { }
-        public Azure.AI.MetricsAdvisor.Models.DimensionKey SeriesGroupKey { get { throw null; } }
+        public MetricSeriesGroupDetectionCondition(Azure.AI.MetricsAdvisor.Models.DimensionKey seriesGroupKey) { }
+        public Azure.AI.MetricsAdvisor.Models.DimensionKey SeriesGroupKey { get { throw null; } set { } }
```

Comment:
Original behavior:
```cs
var condition = new MetricSeriesGroupDetectionCondition();
condition.SeriesGroupKey.AddColumn("name", "value");
condition.SeriesGroupKey.AddColumn("name2", "value2");
```
New behavior:
```cs
var allColumns = new Dictionary<string, string>() { {"name", "value"}, ... };
var condition = new MetricSeriesGroupDetectionCondition(allColumns);
```

Added setter so we can easily change it during Update.